velox/serializers/PrestoSerializer.cpp

```diff
@@ -679,10 +735,12 @@ void writeInt64(OutputStream* out, int64_t value) {
 class VectorStream {
  public:
   VectorStream(
+      bool useLosslessTimestamp,
```

Comment:
nit: shall we put useLosslessTimestamp the last parameter as type shall be the first one logically?

Reply:
done